### ISSUE TITLE
Shop Sorting Implementation

### ISSUE THREAD

**Issue by CristinaSolana** [envistaInteractive/itagroup-ecommerce-template#425]
Payload should be converted and added to query string like others

`sort=`

```
[
    {
        "field": "pricing.active.regular",
	"order": "asc"
    }
]
```

**Comment by itsDiwaker** [comment]
@CristinaSolana 
what should be the url format for example if product listing has been sorted using price in ascending order?

**Comment by CristinaSolana** [comment]
@itsDiwaker when sorting by product.name, we will need to pass product.name.keyword instead.

**Comment by CristinaSolana** [comment]
Also, though it is being sorted, the sort by list does not show "price" selected...

![screen shot 2018-01-02 at 3 46 51 pm](https://user-images.githubusercontent.com/524896/34499417-fcaa0c86-efd3-11e7-903e-e1d0aa92d986.png)

**Comment by itsDiwaker** [comment]
Currently active sort is being shown like this:

<img width="465" alt="screen shot 2018-01-04 at 4 15 08 am" src="https://user-images.githubusercontent.com/10872098/34543219-de7b0430-f105-11e7-8520-e43c606682d2.png">

**Comment by CristinaSolana** [comment]
@itsDiwaker Yes, but if it is being sorted, it should show that when the dropdown is closed.
